package.json

```diff
@@ -15,8 +15,7 @@
   },
   "dependencies": {
     "airplay-js": "guerrerocarlos/node-airplay-js",
-    "application-config": "^0.2.0",
-    "application-config-path": "^0.1.0",
+    "application-config": "feross/node-application-config",
```

Comment:
+1

The original was janky.

If you've already made your own module, you can also use `JSON.stringify(obj, null '\t')` to produce neatly printed configuration files. (Our current config.jsons are all on one line.)